chaos_monkey/config.py

```diff
@@ -178,15 +180,23 @@ def as_dict(self):
 
     range_class = FixedRange
     range_size = 1
-    range_fixed = ['',]
+    range_fixed = ['', ]
 
     blocked_ips = ['', ]
 
     # TCP Scanner
     HTTP_PORTS = [80, 8080, 443,
                   8008,  # HTTP alternate
                   ]
-    tcp_target_ports = [22, 2222, 445, 135, 3389]
+    tcp_target_ports = [22,
```

Comment:
Why remove 2222? Should it not have been there originally?

Reply:
Fixed, my bad.